### PR TITLE
add the network name as well on the storage key prefix 

### PR DESCRIPTION
**Pull request by anandrgitnirman**
#188 Add the network name as part of the storage key prefix
With the single channel per Organization per group , the existing key should suffice ( we dont need to

### CODE REVIEW

**Comment by anandrgitnirman** [comment]
we will need to enforce the configuration below is mandatory 
blockchain_network_selected

File: escrow/payment_channel_storage.go [hunk 1]
```diff
@@ -27,7 +28,8 @@ func NewPaymentChannelStorage(atomicStorage AtomicStorage) *PaymentChannelStorag
 		delegate: &TypedAtomicStorageImpl{
 			atomicStorage: &PrefixedAtomicStorage{
 				delegate:  atomicStorage,
-				keyPrefix: "/payment-channel/storage",
+				//Add the Network as the prefix on the key for storage
```
Comment (raamb) [inline escrow/payment_channel_storage.go]: What happens if BlockChainNetworkSelected is not set in the config? If I remember correct user can set the registry address directly and not need to set this correct?

Comment (anandrgitnirman) [reply escrow/payment_channel_storage.go]: the configurtion is now forced and defualt value has been removed , without this config Daemon will not start

Comment (raamb) [reply escrow/payment_channel_storage.go]: We shouldnt allow the registryaddress and the networkname to be provided as that can lead to incorrect scenarios.

Comment (anandrgitnirman) [reply escrow/payment_channel_storage.go]: @raamb as discussed , we will now use the MPE address as the storage prefix

**Pull request merged**